common/actions/paritySigner/actionTypes.ts

```diff
@@ -1,9 +1,19 @@
 import { TypeKeys } from './constants';
 
-export interface RequestSignatureAction {
-  type: TypeKeys.PARITY_SIGNER_REQUEST_SIGNATURE;
+export interface RequestTransactionSignatureAction {
+  type: TypeKeys.PARITY_SIGNER_REQUEST_TX_SIGNATURE;
   payload: {
-    rlp: string;
+    isMessage: false;
+    data: string;
+    from: string;
+  };
+}
+
+export interface RequestMessageSignatureAction {
+  type: TypeKeys.PARITY_SIGNER_REQUEST_MSG_SIGNATURE;
+  payload: {
+    isMessage: true;
```

Comment:
This is fine for now, but if there's ever another signature type, we should make this an enum, e.g. `payload.type === SIGNATURE_DATA.MESSAGE` or something.